src/backend/storage/file/fd.c

```diff
@@ -2603,13 +2621,20 @@ closeAllVfds(void)
 
 
 /*
- * SetTempTablespaces
+ * SetTempTablespaces & SetFileTempTablespaces
```

Comment:
I would prefer not to combine the comments from two functions in one place, but to place the comments directly before each function.